main/TMP1075.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include "esp_log.h"
+#include "driver/i2c.h"
```

Comment:
include i2c_master.h instead of driver/i2c.h (need to be based on latest master of main repo)